.github/workflows/lint-renovate-pr.yml

```diff
@@ -0,0 +1,70 @@
+name: Lint and Update renovate PR
```

Comment:
same comment as on the `echo` below. Let's name this like `Regenerate docs and helm tests`